gatsby_fp/src/components/Layout.js

```diff
@@ -76,10 +78,17 @@ export const facultyData = graphql`
       website
       research
       room
-      photo_base64
```

Comment:
I think removing this means we should also remove it from line 36 of `gatsby-config.js`